units/src/amount.rs

```diff
@@ -864,6 +864,22 @@ impl Amount {
     /// Returns [None] if overflow occurred.
     pub fn checked_rem(self, rhs: u64) -> Option<Amount> { self.0.checked_rem(rhs).map(Amount) }
 
+    /// Unchecked addition.
+    /// This is the unsafe variant of addition.  Overflow instead of panic.
+    pub fn unchecked_add(self, rhs: Amount) -> Amount {
+        Self(self.0 + rhs.0)
+    }
+
+    /// Unchecked subtraction.
+    /// This is the unsafe variant of subtraction.  Overflow instead of panic.
```

Comment:
It will panic in debug mode. Another reason to consider `raw_` instead of `unchecked_` - it will be checked, sometimes. ;D

Reply:
`raw_` is undiscoverable.

Reply:
Took forever for someone to report needing it, so I don't think it will be (or should be) commonly used thing. So maybe it's not a problem.

Also I don't really see how `raw_xyz` is less discoverable than `anythingelse_xyz`.

If discoverability is really needed (I doubt it) the `#[doc(alias="add_raw")]` could be maybe added to mitigate?

Reply:
I have no idea what `raw_add` does based on the name. If I'm looking for an addition function even if I see that I won't consider using it unless I'm really stuck.

Reply:
Personally if I saw `raw` I wouldn't intuit what is meant.  `unchecked` on the other hand I'd have a good idea.